KubeArmor/BPF/system_monitor.c

```diff
@@ -273,6 +273,35 @@ BPF_PERCPU_ARRAY(bufs_offset, u32, 3);
 
 BPF_PERF_OUTPUT(sys_events);
 
+// == Visibility == //
+
+const u32 file_probe = 0;
+const u32 process_probe = 1;
+const u32 network_probe = 2;
+const u32 caps_probe = 3;
+
+const u32 trace_syscall = 0;
+const u32 ignore_syscall = 1;
+
```

Comment:
Should we perhaps use macro constants/ enums here, since that is the pattern with other constants? WDYT?